native/SQLiteNative_JNI.c

```diff
@@ -143,19 +150,26 @@ Java_io_liteglue_SQLiteNative_sqlc_1db_1open__Ljava_lang_String_2I(JNIEnv *env,
   if ( NULL != filename ) {
     (*env)->ReleaseStringUTFChars(env, filename, _strchars_filename);
   }
-  return _res;
+
+  jclass class = (*env)->FindClass(env,"io/liteglue/SQLiteResponse");
```

Comment:
Maybe we can refactor this code to avoid duplication?

Reply:
Sure, that will be a good improvement.